mmdet/apis/ote/extension/utils/hooks.py

```diff
@@ -131,7 +131,7 @@ def before_run(self, runner):
         self.time_monitor.total_epochs = total_epochs
         self.time_monitor.train_steps = runner.max_iters // total_epochs
         self.time_monitor.steps_per_epoch = self.time_monitor.train_steps + self.time_monitor.val_steps
-        self.time_monitor.total_steps = math.ceil(self.time_monitor.steps_per_epoch * total_epochs)
+        self.time_monitor.total_steps = max(math.ceil(self.time_monitor.steps_per_epoch * total_epochs), 1)
```

Comment:
How is that possible that the result is < 1?

Reply:
Yes, our optimization partners said that sometimes their system tells that there is no need to do any fine-tuning step after optimization, so train_steps = 0. But 'after_epoch' callback is called anyway [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/epoch_based_runner.py#L130) that causes division by 0 while computing progress.